github/commitstatus.go

```diff
@@ -31,6 +32,12 @@ func (e *StatusError) Error() string {
 // API is the GitHub API endpoint.
 const API = "https://api.github.com"
 
+// Maximum number of retries for the retryable http request
+const maxRetries = 3
+
+// Maximum time to wait before next attempt
```

Comment:
The comment is wrong. Should we have to sleep for more than that we would not and just give up so there is no next attempt.

Reply:
Changed...